marimo/_plugins/ui/_impl/input.py

```diff
@@ -326,6 +326,9 @@ class text(UIElement[str, str]):
     - `placeholder`: placeholder text to display when the text area is empty
     - `kind`: input kind, one of `"text"`, `"password"`, `"email"`, or `"url"`
         defaults to `"text"`
+    - `max_length`: maximum length of input
+    - `min_length`: minimum length of input
```

Comment:
When I test, `min_length` has no (visible) effect on the frontend, and no effect on Python either. Values less than the min length are still sent to Python, with or without forms. Testing on Chrome.

If someone makes a text input with min length, say 8, but types in only 3 characters, what will `text.value` be? I guess it might be the empty string, but that's kind of awkward because `""` has length 0, ha ...

Maybe `min_length` doesn't make as much sense as `max_length`, and can just be validated in Python by the developer? In addition to UX, `max_length` is helpful for the developer to prevent users from OOM-ing the kernel by typing in an absurdly long string. Developers will have to handle min_length validation in Python, but at least strings that are too short won't have any performance issues.

Given these considerations, my vote is to just not have a `min_length` option.